core/storage/base_model/gae_models.py

```diff
@@ -1078,6 +1079,11 @@ class BaseSnapshotMetadataModel(BaseModel):
     # A sequence of commands that can be used to describe this commit.
     # Represented as a list of dicts.
     commit_cmds = ndb.JsonProperty(indexed=False)
+    # The user ids that are in some field in commit_cmds.
+    commit_cmds_user_ids = ndb.StringProperty(repeated=True, indexed=True)
+    # The user ids that are enclosed inside content field in the relevant
```

Comment:
... inside the "content" field of the ...

Reply:
Done.